packages/widgets/Source/Geocoder/GeocoderViewModel.js

```diff
@@ -123,6 +123,9 @@ function GeocoderViewModel(options) {
     const destination = data.destination;
     clearSuggestions(that);
     that.destinationFound(that, destination);
+
+    //This treats clicking a suggestion the same as hitting the enterKey
+    that._searchCommand();
```

Comment:
This may technically work but I don't think it's the correct solution. Specifically I'm worried about the circular call of `activateSuggestion -> searchCommand -> activateSuggestion`. I think a more accurate change would be to alter the click handler on suggestions to behave the same as hitting enter and calling `search`.